.github/workflows/pr-code-sniff.yml

```diff
@@ -49,7 +48,4 @@ jobs:
             - name: Run PHPCS
               if: steps.changed-files.outputs.any_changed == 'true'
               working-directory: plugins/woocommerce
-              run: |
-                  HEAD_REF=$(git rev-parse HEAD)
-                  git checkout $HEAD_REF
-                  phpcs-changed -s --git --git-base ${{ github.base_ref }} ${{ steps.changed-files.outputs.all_changed_files }}
+              run: phpcs-changed -s --git --git-base ${{ github.event.pull_request.base.sha }} ${{ steps.changed-files.outputs.all_changed_files }}
```

Comment:
`github.base_ref` was giving `trunk` but that requires a branch that exists in the clone that doesn't exist. This uses the base branch SHA ref which is always present in this clone.